samples/features/sql-big-data-cluster/deployment/private-aks/scripts/deploy-private-aks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+#Define a set of environment variables to be used in resource creations.
+
+export REGION_NAME=northeurope
```

Comment:
Don't you need a subscription ID for this script?

Reply:
Nope.  Cuz it used by URD only as such : 
az network route-table route create -g $RESOURCE_GROUP --name $FWROUTE_NAME --route-table-name $FWROUTE_TABLE_NAME --address-prefix 0.0.0.0/0 --next-hop-type VirtualAppliance --next-hop-ip-address $FWPRIVATE_IP --subscription $SUBID

Reply:
Exactly you need the sub id (+ region and resource group name) to be passed in by the user